.packit.yaml

```diff
@@ -27,6 +31,13 @@ packages:
     specfile_path: python-copr-common.spec
     files_to_sync:
       - python-copr-common.spec
+    actions:
+      create-archive:
+        - bash -c "tito build --tgz --test -o ."
+        - bash -c "ls -1t ./*.tar.gz | head -n 1"
```

Comment:
This seems to leak to "global" config, because also `python-copr` is affected, not just `python-copr-common`. @lbarcziova fyi

Reply:
oh, can you provide some more info on where I can see this, please?

I had a look into our logs and from there, it looks ok:

```
Package config before loading:
{
    "upstream_project_url": "https://github.com/fedora-copr/copr.git",
    "srpm_build_deps": [
        "wait-for-copr"
    ],
    "packages": {
        "python-copr": {
            "downstream_package_name": "python-copr",
            "upstream_package_name": "copr",
            "paths": [
                "./python"
            ],
            "specfile_path": "python-copr.spec",
            "files_to_sync": [
                "python-copr.spec"
            ],
            "upstream_tag_template": "python-copr-{version}",
            "upstream_tag_include": "^python-copr-.*",
            "actions": {
                "post-upstream-clone": [
                    "sh -c \"wait-for-copr --owner $COPR_OWNER --project $COPR_PROJECT python-copr-common `git rev-parse --short HEAD`\""
                ]
            }
        },
        "python-copr-common": {
            "downstream_package_name": "python-copr-common",
            "upstream_package_name": "copr-common",
            "paths": [
                "./common"
            ],
            "specfile_path": "python-copr-common.spec",
            "files_to_sync": [
                "python-copr-common.spec"
            ],
            "actions": {
                "create-archive": [
                    "bash -c \"tito build --tgz --test -o .\"",
                    "bash -c \"ls -1t ./*.tar.gz | head -n 1\""
                ],
                "get-current-version": [
                    "bash -c \"git describe --match python-copr-common-[0-9]* --abbrev=0 HEAD | egrep -o [0-9]+\\.[0-9]+\""
                ]
            },
            "upstream_tag_template": "python-copr-common-{version}",
            "upstream_tag_include": "^python-copr-common-.*"
        },
        ...
```

Reply:
Sorry, I probably wrongly interpreted the logs! :(